linalg/batched/solver.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2010-2024, Lawrence Livermore National Security, LLC. Produced
```

Comment:
According to my editor, this header is included only by `solver.cpp`. I suspect the idea is to always make it available, i.e. to be included by `mfem.hpp`, e.g. via `linalg/linalg.hpp`.

Reply:
Done in 2a1cce46639700b0f173d95941abe9ef4de3940d.